src/plugins/breakpoint.js

```diff
@@ -8,6 +8,7 @@ module.exports = plugin(
       '[data-breakpoint]::after, [data-breakpoint]::before': {
         zIndex: -999,
```

Comment:
We can remove all the other properties, as they were used to hide the pseudo-element:

```css
[data-breakpoint]::after,
[data-breakpoint]::before {
  display: none;
}
```